src/NewCommand.php

```diff
@@ -193,7 +193,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $output->writeln('');
             }
 
-            $output->writeln('  <bg=blue;fg=white> INFO </> Application ready! <options=bold>Build something amazing.</>'.PHP_EOL);
+            $output->writeln("  <bg=blue;fg=white> INFO </> Application ready in <options=bold>[{$name}]</>.".PHP_EOL);
```

Comment:
Still feel that the `Build something amazing!` could be the same line. Something like this:

```
[INFO] Application ready in [./my-application]. Build something amazing!
```